paddle/scripts/submit_local.sh.in

```diff
@@ -112,13 +112,13 @@ fi
 
 case "$1" in
     "train")
-        ${DEBUGGER} $MYDIR/../opt/paddle/bin/paddle_trainer ${@:2}
+        ${DEBUGGER} $MYDIR/../../opt/paddle/bin/paddle_trainer ${@:2}
```

Comment:
Just for your reference:
I build and installed with `root`, then `pip install whl`, and got `paddle` in `/usr/bin` and the binaries are in `/usr/lib64/python2.7/site-packages/usr/local/opt/paddle/bin/`.

Since `paddle` is installed with `scripts`, however other binaries are installed with `data_files` in `setup.py.in`.
I thought maybe them could be installed with the same way, then it should be much easier to find the correct binaries.